README.md

```diff
@@ -111,14 +109,16 @@ export TNF_NON_INTRUSIVE_ONLY=false
 ```
 
 ### Specifiy the location of the partner repo
-This env var is mandatory.
-You must clone the partner [repo](https://github.com/test-network-function/cnf-certification-test-partner)
-and set TNF_PARTNER_SRC_DIR to point to it.
+This env var is optional, but highly recommended if running the test suite from a clone of this github repo. It's not needed or used if running the tnf image.
+
+To set it, clone the partner [repo](https://github.com/test-network-function/cnf-certification-test-partner) and set TNF_PARTNER_SRC_DIR to point to it.
 
 ```shell script
 export TNF_PARTNER_SRC_DIR=~/code/cnf-certification-test-partner
```

Comment:
will it be better to use  export TNF_PARTNER_SRC_DIR=/local_cnf-certification-test-partner_repo_folder/ ?